src/Acr.UserDialogs.Android/Builders/DatePromptBuilder.cs

```diff
@@ -64,5 +69,12 @@ public static long ToUnixTimestamp(this DateTime dateTime)
             var ms = utc.Subtract(Epoch).TotalMilliseconds;
             return Convert.ToInt64(ms);
         }
+
+        private static bool IsBrokenSamsungDevice()
+        {
+            return Android.OS.Build.Manufacturer.Equals("samsung", StringComparison.OrdinalIgnoreCase) &&
+                   Android.OS.Build.VERSION.SdkInt >= BuildVersionCodes.Lollipop &&
+                   Android.OS.Build.VERSION.SdkInt <= BuildVersionCodes.Lollipop;
```

Comment:
Should actually be SdkInt <=LOLLIPOP_MR1, but there is now a adifferent way of resolving this issue as author implemented missing android style parametar in date picker config.